.github/workflows/delete_doc.yml

```
@@ -3,9 +3,14 @@ name: Documentation Removal
 on:
   pull_request_target:
     types: [closed, removed]
+permissions:
+  contents: read
+
 jobs:
   build:
 
+    permissions:
+      contents: write  # for Git to git push
```

Actually, for that job, we use another Github token, setup by the secrets, to push to another repository of the same organization.

```suggestion
    permissions:
      contents: read  # for Git to git push
```